pkg/schema/schema.go

```diff
@@ -201,6 +201,21 @@ func (t *Table) RenameColumn(from, to string) {
 	delete(t.Columns, from)
 }
 
+func (t *Table) RenameConstraint(from string, to string) {
+	if _, ok := t.CheckConstraints[from]; ok {
+		t.CheckConstraints[to] = t.CheckConstraints[from]
+		delete(t.CheckConstraints, from)
+	}
+	if _, ok := t.UniqueConstraints[from]; ok {
+		t.UniqueConstraints[to] = t.UniqueConstraints[from]
+		delete(t.UniqueConstraints, from)
+	}
+	if _, ok := t.ForeignKeys[from]; ok {
+		t.ForeignKeys[to] = t.ForeignKeys[from]
+		delete(t.ForeignKeys, from)
+	}
```

Comment:
I think it's fine to rename the different constraint places we have in the schema, but this is the part I had more doubts. Please double check 😄

Reply:
This function isn't needed as there is no benefit to renaming the constraint in the virtual schema that gets built during migration start.

Its only purpose is to be able generate views with columns mapped to the right columns on the underlying table. Any constraints on the virtual schema aren't used for anything.